support-frontend/assets/components/marketingConsent/marketingConsentWithCheckbox.jsx

```diff
@@ -92,6 +93,23 @@ function MarketingButton(props: ButtonPropTypes) {
   );
 }
 
+const createRegionalCopy = (titleFreq: string, msgFreq: string): { titleFreq: string, msgFreq: string } => ({
+  titleFreq,
+  msgFreq,
+});
+
+const getCopyTexts = (countryGroupId: CountryGroupId, contributionType: ContributionType) => {
+  const copy = {
```

Comment:
I'd move this outside the function since it's not dependent on it

Reply:
👍